kani-driver/src/cbmc_output_parser.rs

```diff
@@ -1032,3 +1116,106 @@ fn determine_verification_result(properties: &[Property]) -> bool {
         properties.iter().filter(|prop| prop.status == CheckStatus::Failure).count();
     number_failed_properties == 0
 }
+
+#[test]
```

Comment:
That's awesome! Can you please wrap these with a test module?
```
#[cfg(test)]
mod tests {
}

Reply:
Yes, done.